src/server.cpp

```diff
@@ -989,6 +1001,26 @@ JitterMeas.Measure();
     // one client is connected.
     if ( iNumClients > 0 )
     {
+
+        // Low frequency updates
+        if ( iFrameCount > CHANNEL_LEVEL_UPDATE_TIME )
+        {
+            iFrameCount = 0;
+
+            // Enable channel levels if any client has requested them
+            for ( int i = 0; i < iNumClients; i++ )
+            {
+                if ( vecChannels[ vecChanIDsCurConChan[i] ].ChannelLevelsRequired() )
+                {
+                    emit AudioFrames ( static_cast<int16_t>(iNumClients),
```

Comment:
That is not gonna work correctly. I tried your version out on my laptop and I saw that if I made a short noise, the level meter would not show any input. So what you do here is to measure the level at an arbitrary instant in time. But what you would have to do is to measure the level continuously and smooth it by some filter operation and then pick the current level at the channel_level_update_time. Of course this requires additional calculations at the server but this would be the right way of doing it.

Reply:
True enough but it's not intended to be perfect, only a rough indicator that there's some degree of activity.  A short burst isn't enough to do that.  It's only tracking one in 200 frames, for instance, so any burst could be missed entirely.

Reply:
Ok, I would have to see it how it performs if many clients are connected and we have multiple mixer faders. BTW, I saw that the new LED bar next to the fader did have more than just three LEDs. Why did you use a larger number of LEDs?

Reply:
I've added smoothing in the latest commit.

Reply:
Thinking about it... We're currently using 2 bits per level 0 to 3; we could get 0 to 15 by using 4 bits per level, which would be "full precision" for the level meter.  (Easier than 3 bits...)

Reply:
Ok, you can try it if you like. 4 bits should not be too bad in terms of network load.